include/powsybl/iidm/Connectable.hpp

```diff
@@ -22,7 +21,7 @@ namespace iidm {
 
 class Network;
 
-class Connectable : public Identifiable, public MultiVariantObject {
+class Connectable : public Identifiable {
```

Comment:
Update the PR description to report that you're backporting #682

You forget to change the implementation of the MultiVariantObject methods to call the identifiable code. MultiVariant extensions of HvdcLine will not be notified by a variant change